config/settings.yml

```diff
@@ -20,3 +20,10 @@ workflow_url: 'https://workflow.example.edu/'
 dor_services:
   url: 'http://localhost:3003'
   token: 'not-a-real-token'
+
+
+rabbitmq:
+  hostname: localhost
+  vhost: /
+  username: guest
+  password: guest
```

Comment:
Looks good. I see that QA and Stage in shared_configs are configured slightly differently. Looks like one has a vhost and one doesn't. Intended? And I assume doing this for prod is in your plan later on.

Reply:
It's a PR awaiting review: https://github.com/sul-dlss/shared_configs/pull/1723